### PR TITLE
Newsletter-90:Translate into Japanese

### PR DESCRIPTION
**Pull request by Ichiro0219**

### CODE REVIEW

File: _posts/ja/newsletters/2020-03-25-newsletter-ja.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [なぜbanscoreはデフォルトで100になっているのですか？]({{bse}}93795)匿名ユーザは`banscore`の背景にある歴史について説明しています。`banscore`は不正な動きを行うピアからノードを守るためのものです。いくつかの違反行為は100ポイント増加となり、該当ピアはデフォルトの`banscore`の設定配下では即座に追放されますが、[`net_processing.cpp`][bitcoin net processing]で詳細に説明されているその他の違反行為は、それぞれスコアが異なります。
+
+- [ブロック620826のタイムスタンプがブロック620825の1秒前なのはなぜですか？]({{bse}}93696)Andrew Chow氏とRaghav Sood氏は、ブロックヘッダのタイムスタンプ・フィールドが以前のブロックよりも大きな値を持つ必要はないことを説明しています。ただし、新しいブロックのタイムスタンプは、過去11ブロックの中央値よりも大きくなければならず、ノードを実行しているコンピュータの時計ベースで現在時刻から2時間以降になってはならないという要件があります。
+
+- [ミニスクリプトポリシー言語仕様書はどこにありますか？]({{bse}}93764)Andrew ChowとPieter Wuilleは、[ミニスクリプト][topic miniscript]ポリシー言語がどのようにしてミニスクリプトにコンパイルされるかについては仕様がなく、現在のC++とRustの実装では、事実上あらゆる可能性を試した結果として最小の`scriptWitness`サイズになるminiscriptを選択していると説明しています。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2020-03-25-newsletter-ja.md]: For terminology and auto-anchor plugin, it is better 
s/ミニスクリプト/minisctipt/
#349

Comment (Ichiro0219) [reply _posts/ja/newsletters/2020-03-25-newsletter-ja.md]: Thank you @AkioNak , I Fixed it.

File: _posts/ja/newsletters/2020-03-25-newsletter-ja.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+title: 'Bitcoin Optech Newsletter #90'
+permalink: /ja/newsletters/2020/03/25/
+name: 2020-03-25-newsletter
```
Comment (AkioNak) [inline _posts/ja/newsletters/2020-03-25-newsletter-ja.md]: 2020-03-25-newsletter-ja ?

File: _posts/ja/newsletters/2020-03-25-newsletter-ja.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+*[Bitcoin StackExchange][bitcoin.se]はOptech Contributor達が疑問に対して答えを探しに（もしくは他のユーザーの質問に答える時間がある場合に）アクセスする、数少ない情報ソースです。この月刊セクションでは、前回アップデート以降にされた、最も票を集めた質問・答えを紹介しています。*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [なぜbanscoreはデフォルトで100になっているのですか？]({{bse}}93795)匿名ユーザは`banscore`の背景にある歴史について説明しています。`banscore`は不正な動きを行うピアからノードを守るためのものです。いくつかの違反行為は100ポイント増加となり、該当ピアはデフォルトの`banscore`の設定配下では即座に追放されますが、[`net_processing.cpp`][bitcoin net processing]で詳細に説明されているその他の違反行為は、それぞれスコアが異なります。
+
+- [ブロック620826のタイムスタンプがブロック620825の1秒前なのはなぜですか？]({{bse}}93696)Andrew Chow氏とRaghav Sood氏は、ブロックヘッダのタイムスタンプ・フィールドが以前のブロックよりも大きな値を持つ必要はないことを説明しています。ただし、新しいブロックのタイムスタンプは、過去11ブロックの中央値よりも大きくなければならず、ノードを実行しているコンピュータの時計ベースで現在時刻から2時間以降になってはならないという要件があります。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2020-03-25-newsletter-ja.md]: Same as a line below, I think that it has not usually used titles such as "氏" in this translated version of the newsletter.

File: _posts/ja/newsletters/2020-03-25-newsletter-ja.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+- [miniscriptポリシー言語仕様書はどこにありますか？]({{bse}}93764)Andrew ChowとPieter Wuilleは、[miniscript][topic miniscript]ポリシー言語がどのようにしてminiscriptにコンパイルされるかについては仕様がなく、現在のC++とRustの実装では、事実上あらゆる可能性を試した結果として最小の`scriptWitness`サイズになるminiscriptを選択していると説明しています。
+
+## Notable code and documentation changes
+
+*今週の[Bitcoin Core][bitcoin core repo]、[C-Lightning][c-lightning repo]、[Eclair][eclair repo]、[LND][lnd repo]、[libsecp256k1][libsecp256k1 repo]、[Bitcoin Improvement Proposals (BIPs)][bips repo]および[Lightning BOLTs][bolts repo]の注目すべき変更点。*
+
+- [Eclair #1339][]では、ユーザがhtlc-minimumを0 milli-satoshisに設定することを防ぎます。これは[BOLT2][]<!-- "A
+  receiving node [...] receiving an `amount_msat` equal to 0 [...]
+SHOULD fail the channel." -->に違反します。新しいhtlc-minimum設定値は0 milli-satoshisとなります。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2020-03-25-newsletter-ja.md]: s/0/1/

Comment (Ichiro0219) [reply _posts/ja/newsletters/2020-03-25-newsletter-ja.md]: @AkioNak Fix done.

File: _posts/ja/newsletters/2020-03-25-newsletter-ja.md (hunk 3 above)
Comment (Ichiro0219) [reply _posts/ja/newsletters/2020-03-25-newsletter-ja.md]: @AkioNak Fix done.

File: _posts/ja/newsletters/2020-03-25-newsletter-ja.md (hunk 2 above)
Comment (Ichiro0219) [reply _posts/ja/newsletters/2020-03-25-newsletter-ja.md]: @AkioNak Fix done.

**Pull request merged**